From 6a3025a0f47c568cbc06cd7e9a3b3d744d4d3545 Mon Sep 17 00:00:00 2001
From: Amir Vadai <avadai@redhat.com>
Date: Mon, 3 Nov 2014 07:52:34 -0500
Subject: [infiniband] iser: Bump version to 1.4.1

Message-id: <261ab9cf5585d30e30694a0ac0090e091fceccfb.1415000301.git.avadai@redhat.com>
Patchwork-id: 98705
O-Subject: [Patch RHEL7.1 36/37] IB/iser: Bump version to 1.4.1
Bugzilla: 1107622
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Doug Ledford <dledford@redhat.com>

From: Or Gerlitz <ogerlitz@mellanox.com>

Signed-off-by: Roi Dayan <roid@mellanox.com>
Signed-off-by: Or Gerlitz <ogerlitz@mellanox.com>
Signed-off-by: Roland Dreier <roland@purestorage.com>
(cherry picked from commit 61aabb3c91c1b03478ffc1a4a2573f825e7f35f9)
Signed-off-by: Amir Vadai <amirv@mellanox.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/infiniband/ulp/iser/iscsi_iser.h b/drivers/infiniband/ulp/iser/iscsi_iser.h
index c877dad..9f0e0e3 100644
--- a/drivers/infiniband/ulp/iser/iscsi_iser.h
+++ b/drivers/infiniband/ulp/iser/iscsi_iser.h
@@ -69,7 +69,7 @@
 
 #define DRV_NAME "iser"
 #define PFX  DRV_NAME ": "
-#define DRV_VER  "1.4"
+#define DRV_VER  "1.4.1"
 
 #define iser_dbg(fmt, arg...)    \
  do {      \
-- 
1.7.1